### DISCUSSION TITLE
@jesus christ!!! Unacceptable Disruption by AdGuard: Immediate Rectification Imperative

### DISCUSSION THREAD

**Discussion by jameschung0225** [List-KR/community#23] [General]
AdGuard service has created an untenable situation by causing a total shutdown of a crucial banking application in Korea. This has occurred due to your service obstructing the DNS resolution for the domain mbp.hanabank.com. 

This is a gross violation of the reliability expected from your service and is absolutely insupportable. I do not merely request but insist upon immediate and decisive action to rectify this flagrant issue. This is a matter impacting financial activities of countles

**Comment by piquark6046** [comment]
Duplicate of List-KR/community#17
